src/HuntAndPeck/Views/OverlayView.xaml

```diff
@@ -43,13 +43,13 @@
             <ItemsControl.ItemTemplate>
                 <DataTemplate>
                     <Grid>
-                        <Rectangle Width="{Binding Hint.BoundingRectangle.Width}" Height="{Binding Hint.BoundingRectangle.Height}" Stroke="Red" StrokeDashArray="1 2">
+                        <!--<Rectangle Width="{Binding Hint.BoundingRectangle.Width}" Height="{Binding Hint.BoundingRectangle.Height}" Stroke="Red" StrokeDashArray="1 2">
```

Comment:
Can you please remove this?

Reply:
There's a file conflict, normally I wouldn't worry about commented code :)

Reply:
I resolved a conflict in `HintLabelService.cs`. There's no conflict now..

Reply:
Thanks :+1: